app/src/main/java/space/taran/arknavigator/ui/fragments/GalleryFragment.kt

```diff
@@ -314,10 +330,11 @@ class GalleryFragment : MvpAppCompatFragment(), GalleryView, NotifiableView {
 
     private fun setupOpenEditFABs(kind: ResourceKind?) {
```

Comment:
maybe it's a good time to start adding some documentation to the method definition. Maybe not all of the methods but methods which we thing are not good enough named for other devs. `setupOpenEditFABs`  doesn't really describe the method just right in my opinion. What does `FAB` mean for example?

Reply:
Maybe we should create a separate issue for this, but it seems to me that there will still be refactoring.
FAB stands for [floating action button](https://material.io/components/buttons-floating-action-button/android)

Reply:
I would name this action in more generic way, like `setupResourceActions()` or `setupResourceExternalOperations()` or `displayControls()`.